data/state/account_test.go

```diff
@@ -242,3 +243,104 @@ func TestAccount_SetCodeHashWithJournal(t *testing.T) {
 	assert.Equal(t, 1, journalizeCalled)
 	assert.Equal(t, 1, saveAccountCalled)
 }
+
+func TestAccount_SetOwnerAddressWithJournal(t *testing.T) {
+	t.Parallel()
+
+	count := 0
```

Comment:
use 2 booleans instead of a common counter. maybe `journalizeWasCalled` and `saveAccountWasCalled`

Reply:
done.